lib/_http_incoming.js

```diff
@@ -349,6 +358,14 @@ IncomingMessage.prototype._dump = function _dump() {
   }
 };
 
+function onError(instance, cb, error) {
+  if (instance.listenerCount('error') > 0) {
```

Comment:
Maybe a comment describing why we do this?

Reply:
Yup, thanks